core/src/main/java/com/spotify/metrics/core/SemanticMetricRegistryListener.java

```diff
@@ -184,4 +132,91 @@ public void onDerivingMeterRemoved(MetricId name) {
      * @param name the meter's name
      */
     void onDerivingMeterRemoved(MetricId name);
+
```

Comment:
@ao2017 could you pls separate this reordering change in diff PR? It is really big one and it's difficult to follow all the changes.

Reply:
Done